bambi/priors/scaler.py

```diff
@@ -114,7 +114,7 @@ def scale_threshold(self):
             threshold = self.model.components["threshold"]
             if isinstance(threshold, ConstantComponent) and threshold.prior.auto_scale:
                 response_level_n = len(np.unique(self.response_component.response_term.data))
-                mu = np.round(np.linspace(-2, 2, num=response_level_n - 1), 2)
+                mu = np.zeros(response_level_n - 1)
```

Comment:
If sequential models assume that for every response level there is a latent continuous variable $Z_k$, then wouldn't we need each response level, i.e., `mu = response_level_n` and **not** `response_level_n  - 1`?

Reply:
Ohh, I think you're right! 

I'm thinking why the current approach is working and not failing. Is it because it's not considering the probability of Y being larger than the largest observed category? I think it would make sense for the years example, but I'm not sure if it would make sense for cases where there is a pre-specified set of categories. 

I wrote that as I was looking at this visualization from the ordinal tutorial by Bürkner and Vuorre:

![image](https://github.com/bambinos/bambi/assets/25507629/879d4dc3-ee52-47a4-ba13-ee815a52c4c2)

and I wonder: Do we always have a Pr(Y > K)?

Reply:
If I remove the `-1`, I get `ValueError: Incompatible Elemwise input shapes [(35,), (36,)]`. This makes sense as I stated in the docs because the sequential model is a product of probabilities, i.e., the probability that $Y$ is equal to category $k$ is equal to the probability that it did not fall in one of the former categories $1: k-1$ multiplied by the probability that the sequential process stopped at $k$.